TESTOWNERS

```diff
@@ -220,9 +221,7 @@
 /dev/devicelab/bin/tasks/flutter_gallery_macos__start_up.dart @cbracken @flutter/desktop
 /dev/devicelab/bin/tasks/flutter_gallery_win_desktop__compile.dart @yaakovschectman @flutter/desktop
 /dev/devicelab/bin/tasks/flutter_gallery_win_desktop__start_up.dart @yaakovschectman @flutter/desktop
-/dev/devicelab/bin/tasks/flutter_tool_startup__linux.dart @jensjoha @flutter/tool
-/dev/devicelab/bin/tasks/flutter_tool_startup__macos.dart @jensjoha @flutter/tool
-/dev/devicelab/bin/tasks/flutter_tool_startup__windows.dart @jensjoha @flutter/tool
```

Comment:
Sorry I forgot to mention earlier, shall we keep these three entries as well to avoid breakage while cocoon is not deployed yet?
We can have a follow up clean up PR to remove them, together with the spell_check_test ones.

Reply:
Yeah we can do that.